mobile/lib/modules/search/views/search_result_page.dart

```diff
@@ -163,15 +154,15 @@ class SearchResultPage extends HookConsumerWidget {
       }
 
       if (searchResultPageState.isSuccess) {
-        if (isDisplayDateGroup.value) {
-          return ImmichAssetGrid(
-            assets: allSearchAssets,
-          );
-        } else {
-          return SearchResultGrid(
-            assets: allSearchAssets,
-          );
-        }
+        return MultiselectGrid(
+          renderListProvider: searchRenderListProvider,
+          archiveEnabled: true,
+          deleteEnabled: true,
+          editEnabled: true,
+          favoriteEnabled: true,
+          stackEnabled: false,
+          onRefresh: refresh,

```

Comment:
This is great. Using the refresh in `MultiSelectGrid` to actually refresh the search